packages/monaco-graphql/README.md

```diff
@@ -516,13 +518,54 @@ some gotchas:
 - "linting" => "diagnostics" in lsp terminology
 - the default keymap is different, more vscode like
 - command palette and right click context menu are important
-- you can extend the standard completion/linting/etc provided. for example,
+- you can extend the standard completion/linting/etc. provided. for example,
   `editor.setModelMarkers()`
 - [Monaco Editor API Docs](https://microsoft.github.io/monaco-editor/api/index.html)
 - [Monaco Editor Samples](https://github.com/Microsoft/monaco-editor-samples)
   repository is great for tips on implementing with different bundlers,
   runtimes, etc.
 
+## Avoid Bundle All `monaco-editor`'s Languages
+
+While importing `monaco-editor` in your project, you silently import 83 builtin
+languages, such as `typescript`, `html`, `css`, `json` and other. You can found
+a full list of
+[basic-languages](https://github.com/microsoft/monaco-editor/tree/main/src/basic-languages)
+and
+[languages](https://github.com/microsoft/monaco-editor/tree/main/src/language).
+
+For `monaco-graphql` basically you need only 2 languages - `graphql` and `json`.
+To improve performance and drop loading unused languages starting from
+monaco-graphql@1.3.0 version you can do it by replacing all your `monaco-editor`
+exports with `monaco-graphql/esm/monaco-editor` that will load only `graphql`
+and `json` languages.
```

Comment:
```suggestion
For `monaco-graphql`, you need only 2 languages - `graphql` and `json`.
In version monaco-graphql@1.3.0 and later, you can replace all `monaco-editor`
exports with `monaco-graphql/esm/monaco-editor` to improve performance, load
only `graphql` and `json` languages, and skip loading unused languages.
```

Reply:
can commit, since changed little but, but applied in my commit, thx